packages/button/components/ButtonBase.tsx

```diff
@@ -54,12 +54,10 @@ export interface ButtonProps extends LinkProps {
   /**
    * The icon that appears before the button text
    */
-  // TODO: only accept IconShapes when we make a big breaking change
   iconStart?: IconShapes | React.ReactElement<HTMLElement>;
```

Comment:
I created a JIRA to follow up and track these two - https://d2iq.atlassian.net/browse/D2IQ-99153